easybuild/easyconfigs/a/Arrow/Arrow-6.0.1-foss-2021a.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '6.0.1'
+
+homepage = 'https://arrow.apache.org'
+description = """Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
+ for in-memory data."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://archive.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s']
+sources = ['apache-arrow-%(version)s.tar.gz']
+checksums = ['3786b3d2df954d078b3e68f98d2e5aecbaa3fa2accf075d7a3a13c187b9c5294']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Autotools', '20210128'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.6'),
+    ('pkg-config', '0.29.2'),
+]
+
+# Arrow strongly prefers included jemalloc, so not including it as a dependency
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('Boost', '1.76.0'),
+]
+
+separate_build_dir = True
+start_dir = 'cpp'
+
+# see https://arrow.apache.org/docs/python/development.html
```

Comment:
This URL is broken, correct link is now https://arrow.apache.org/docs/developers/python.html

Reply:
@ekaterinaasf Please change this comment line in the easyconfig file to use to correct URL:

```suggestion
# see https://arrow.apache.org/docs/developers/python.html